bin/live/pycbc_live_plot_combined_single_fits

```diff
@@ -151,7 +152,7 @@ for ifo in ifos:
         bin_colour = plt.get_cmap(args.colormap)(bin_prop)
         bin_label = f"duration {bl:.2f}-{bu:.2f}"
         alpha_lines += ax_alpha.plot(separate_starts[ifo], alphas, c=bin_colour,
-                                     label=bin_label)
+                                     label=bin_label, marker='x')
```

Comment:
Could you also try with the `.` (period) marker? For some reason crosses make plots look more cluttered to me, and are less visible.